src/coreclr/jit/ee_il_dll.cpp

```diff
@@ -1122,9 +1122,39 @@ void Compiler::eeDispLineInfos()
  * (e.g., host AMD64, target ARM64), then VM will get confused anyway.
  */
 
+void Compiler::eeAllocMem(AllocMemArgs* args, UNATIVE_OFFSET hotSizeRequest, UNATIVE_OFFSET coldSizeRequest)
```

Comment:
This function should have the same signature as `allocMem`: no need to pass `hotSizeRequest` or `coldSizeRequest` since they can be found from `args->hotCodeSize` and `args->coldCodeSize`.